module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <array>
+#include <vector>
+#include <algorithm>
+
+constexpr size_t width = 32;
+constexpr size_t height = 32;
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>& arr);
+std::array<std::array<uint8_t, width>, height> decompressGrayscale(std::vector<std::pair<uint8_t, uint8_t>>);
```

Comment:
Have you considered using aliases for longer types like std::vector<std::pair<uint8_t, uint8_t>>?